.github/workflows/ci.yml

```diff
@@ -120,4 +120,20 @@ jobs:
             make || exit 1; \
             cd ..; \
           done;
-
+  cmake-windows-latest:
+    runs-on: windows-latest
+    env:
+      APACHE_LOUNGE_DISTRO_VERSION: 2.4.57
+      HTTPD_DEV_HOME: '${{ github.workspace }}\httpd-apache-lounge\Apache24'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Developer Command Prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
+      - name: Get httpd
+        run: |
+          curl.exe --output "httpd-2.4.57-win64-VS16.zip" --url "https://www.apachelounge.com/download/VS16/binaries/httpd-2.4.57-win64-VS16.zip"
+          Expand-Archive -Path "httpd-2.4.57-win64-VS16.zip" -DestinationPath "httpd-apache-lounge"
```

Comment:
Can we make these to use the env variable `APACHE_LOUNGE_DISTRO_VERSION`? 3x

Reply:
I tried it before, but it did not work. However it might be a problem (like a typo) on my side now that I'm thinking about it... let's wait and see. (I tried the `commit changes`, but even if it passes, I'll squash it.)

Good catch, thank you.

Reply:
> That didn't work, but `"${{ env.APACHE_LOUNGE_DISTRO_VERSION }}"` did. Also, there is no reason to name the zip locally with such a long name -> thank you, it's even better now.

Good to now - sounds like github is resolving the expression before passing onto windows.